spec/lib/feature_management_spec.rb

```diff
@@ -226,4 +226,26 @@
       end
     end
   end
+
+  describe '#enable_agency_based_uuids?' do
+    context 'when enabled' do
+      before do
+        allow(Figaro.env).to receive(:enable_agency_based_uuids).and_return('true')
+      end
+
+      it 'enables the feature' do
+        expect(FeatureManagement.enable_agency_based_uuids?).to eq(true)
+      end
+    end
+
+    context 'when disabled' do
+      before do
+        allow(Figaro.env).to receive(:enable_agency_based_uuids).and_return('false')
+      end
+
+      it 'disables the feature' do
+        expect(FeatureManagement.enable_agency_based_uuids?).to eq(false)
+      end
+    end
+  end
```

Comment:
Can we add a test for `agencies_with_agency_based_uuids`?

Reply:
Done.